### DISCUSSION TITLE
Each time the plugin tries to delete the images

### DISCUSSION THREAD

**Discussion by traveligion** [oleeskild/obsidian-digital-garden#344] [Q&A]
Each time the plugin tries to delete the images, but nothing happens. When I refresh publish notes again, it tries again to delete a large number of images. It is now about 300 images and although nothing happens, the push process is very long. 

Maybe it has to do with moving notes, but how do I get rid of the long attempt to delete a large number of images every update?

**Comment by chaosraine** [comment]
I'm having this same issue, and it does seem like it has to do with moving files around. I fixed it once before by just deleting all of the images in my repo and adding them back, but it's not really a long term solution. I have 82 images being "deleted" every time I publish
